Cargo.toml

```diff
@@ -1,26 +1,27 @@
 [package]
 name = "wluma"
-version = "4.4.0"
+version = "4.4.1"
 authors = ["Maxim Baz", "Cyril Levis"]
 edition = "2021"
 
 [dependencies]
+ash = { version = "~0.38", features = ["linked"], default-features = false }
 wayland-client = { version = "~0.29", features = ["dlopen"] }
 wayland-protocols = { version = "~0.29", features = ["client", "unstable_protocols"] }
 serde = { version = "~1.0", features = ["derive"] }
 serde_yaml = "~0.9"
 toml = "~0.8"
 chrono = "~0.4"
-ash = { version = "~0.37", features = ["linked"], default-features = false }
 itertools = "~0.13"
 v4l = { version = "~0.14", features = ["libv4l"], default-features = false }
 ddc-hi = "~0.4"
 log = "~0.4"
 env_logger = "~0.11"
-inotify = "~0.10"
-lazy_static = "~1.4"
+inotify = "~0.11"
+lazy_static = "~1.5"
 xdg = "~2.5"
 dbus = "~0.9"
+anyhow = "1.0"
```

Comment:
Let's also use `~1.0` as other deps do - by the way, love the idea of trying this crate out :+1: 

In a separate PR after this is merged, I would totally be open to considering replacing the `std::Result` with `anyhow::Result`, i.e. that all the functions get changed from `Result<T, Box<dyn Error>>` to anyhow's `Result<T>` - should make it a lot easier!